app/models/invitation_to_organization.rb

```diff
@@ -0,0 +1,50 @@
+class InvitationToOrganization < Invitation
+  belongs_to :member, inverse_of: :created_invitations, foreign_key: :invited_by_id
+  belongs_to :organization, inverse_of: :invitations
+
+  validates :invited_by_id, :role, presence: true
+  validate :role_inclusion
+  validate :email_uniq
+
+  delegate :organization, to: :member
+
+  after_create :send_invitation
+
+  enumerize :role, in: [:user, :admin, :owner], default: :user, predicates: true
+
+  def owner
```

Comment:
@dissident rename to `owner_name`. And I think you may make `owner` as association through `member`